packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ModelController.java

```diff
@@ -275,11 +279,11 @@ ResponseEntity<List<ModelConfiguration>> getModelConfigurationsForModelId(
 
 
 				// Find the Document Assets linked via provenance to the model configuration
-				final ProvenanceQueryParam body = new ProvenanceQueryParam();
-				body.setRootId(config.getId());
-				body.setRootType(ProvenanceType.MODEL_CONFIGURATION);
-				body.setTypes(List.of(ProvenanceType.DOCUMENT));
-				final Set<String> documentIds = provenanceSearchService.modelConfigFromDocument(body);
+				final ProvenanceQueryParam documentSearchBody = new ProvenanceQueryParam();
```

Comment:
```suggestion
				final ProvenanceQueryParam queryParam = new ProvenanceQueryParam();
```

Reply:
changed to `documentQueryParams` and `datasetQueryParams`